lib/table/features/date-edit/template-input.html

```diff
@@ -1,18 +1,18 @@
 <div>
-  <h4>Edit Date Condition</h4>
+  <h4>{{ translate('Edit Date Condition') }}</h4>
   <select class="dateEdit-type-dropdown">
-    <option value="exact">Exactly</option>
-    <option value="before">Before</option>
-    <option value="after">After</option>
-    <option value="between">Between</option>
+    <option value="exact">{{ translate('Exactly') }}</option>
```

Comment:
Please remove the `translate(` + `)` parts here and invoke translate from DateEdit.js.